CONTRIBUTING.md

```diff
@@ -515,6 +516,62 @@ this API we will not break anyone, because they weren't able to use it. The file
 `allowed-breaking-changes.txt` in the root of the repo is an exclusion file that
 can be used in these cases.
 
+### Examples
+
+Examples typed in fenced code blocks (looking like `'''ts`, but then with backticks
```

Comment:
You should be able to actually put backticks in code blocks like this -  \<backtick>\<space>\<backtick>\<backtick>\<backtick>ts\<backtick>

Obviously, I'm using \<backtick> and \<space> to preven md rendering here :)

Example - ` ```ts`